storage/storage.go

```diff
@@ -84,3 +84,41 @@ func (st *Storage) CheckFundedUserWithWaitTime(userID []byte, authType string) (
 	wp := binary.LittleEndian.Uint64(wpBytes)
 	return wp >= uint64(time.Now().Unix()), time.Unix(int64(wp), 0)
 }
+
+// AddPendingStripeSession adds a pending Stripe session to the storage.
+// It takes a sessionID string as a parameter and returns an error if any.
+func (st *Storage) AddPendingStripeSession(sessionID string) error {
```

Comment:
I don't really like to have something specific to Stripe in the storage package. I'd just name this methods Add/Get/Del with key, value []byte attributes

Reply:
Agreed. Though since Set y Get already exist, I think the best would be to  remove the stripe specific ones, and implement the Delete function